scopes/react/ui/docs-app/base.tsx

```diff
@@ -50,15 +50,18 @@ export function Base({ docs = defaultDocs, componentId, compositions, renderingC
 
   return (
     <div className={classNames(styles.docsMainBlock)} {...rest}>
-      <ComponentOverview
-        displayName={Content.displayName || displayName}
-        version={version}
-        abstract={description || Content.abstract || abstract}
-        labels={component.labels || Content.labels || labels}
-        packageName={packageName}
-        elementsUrl={elementsUrl}
-      />
-
+      {component.preview?.includesEnvTemplate === false ? (
+        <></>
+      ) : (
```

Comment:
```suggestion
      {component.preview?.includesEnvTemplate !== false && (
```

[critical] make sure this condition makes sense